lib/Doctrine/Template/Geographical.php

```diff
@@ -90,14 +90,14 @@ public function getDistanceQuery()
     /**
      * Get distance between this record and another
      *
-     * @param string $Doctrine_Record 
-     * @param string $kilometers 
+     * @param string $Doctrine_Record
+     * @param string $kilometers
      * @return integer
      */
     public function getDistance(Doctrine_Record $record, $kilometers = false)
     {
-        $query = $this->getDistanceQuery($kilometers);
```

Comment:
Method does not accept a parameter. The `getDistanceQuery` method actually selects both `miles` and `kilometers` from the db and then uses this `$kilometers` variable to decide which to return later (117).